### DISCUSSION TITLE
rip to the fastpass bot

### DISCUSSION THREAD

**Discussion by ethanbrimhall** [ethanbrimhall/Disney-Fastpass-Bot#13] [General]
well it was a fun run, bye fastpass bot

**Comment by joeyman178** [comment]
What happened to it :o

**Reply by albertperk** [reply]
On October 19th, 2021 Walt Disney World announced a new system called Disney Genie+ with Lightning Lane. It's a similar service however you have to pay in order to experience rides from a shorter line, and it also has restrictions on how many selections you can make per day. FastPass+ was free without limitations. 

Now that FastPass+ has been retired and succeeded by Lightning Lane, unfortunately, the FastPass Bot is no longer needed and was basically sunsetted due to its replacement, Lightning Lane. RIP FastPass Bot 😭  Thanks for everything, @ethanbrimhall, this was an excellent bot, and helped me get a few hard to get FastPasses.

**Reply by joeyman178** [reply]
Ahhh, I see. If only there was a script for genie+ and lightning lane. Wouldn't that be amazing? I understand that it's still hard to book!

Disney are a shambles of a company, squeezing their guests for every cent.

**Reply by ethanbrimhall** [reply]
yeah genie+ is built completely different now. i'm unsure if it's even available to book attractions from the web. i really miss the old system with fastpass. wait times seems to be worse and much more of a headache for people to figure out how to use genie+.
